layers/+lang/python/funcs.el

```diff
@@ -217,15 +217,14 @@ ROOT-DIR should be the path for the environemnt, `nil' for clean up"
 
 ;; from https://www.snip2code.com/Snippet/127022/Emacs-auto-remove-unused-import-statemen
 (defun spacemacs/python-remove-unused-imports ()
-  "Use Autoflake to remove unused function"
-  "autoflake --remove-all-unused-imports -i unused_imports.py"
+  "Use Autoflake to remove the unused imports:
+autoflake --remove-all-unused-imports -i <file.py>"
   (interactive)
-  (if (executable-find "autoflake")
-      (progn
-        (shell-command (format "autoflake --remove-all-unused-imports -i %s"
-                               (shell-quote-argument (buffer-file-name))))
-        (revert-buffer t t t))
-    (message "Error: Cannot find autoflake executable.")))
+  (cond ((executable-find "autoflake")
+         (shell-command (format "autoflake --remove-all-unused-imports -i %s"
+                                (shell-quote-argument (buffer-file-name))))
+         (revert-buffer t t t))
+        (t (message "Error: Cannot find autoflake executable."))))
```

Comment:
```suggestion
        (t (user-error "Cannot find autoflake executable"))))
```

Reply:
Good idea. And included in the latest commit. Thanks.